Description: Set the bookmarks number before submenu to 8 which should let users configure some before hitting the limit
Bug: https://bugs.launchpad.net/ubuntu/+source/gnome-panel/+bug/204567 

Index: gnome-panel-2.29.5.1/gnome-panel/panel-menu-items.c
===================================================================
--- gnome-panel-2.29.5.1.orig/gnome-panel/panel-menu-items.c	2010-01-15 00:10:43.000000000 +0100
+++ gnome-panel-2.29.5.1/gnome-panel/panel-menu-items.c	2010-01-15 00:10:56.000000000 +0100
@@ -62,7 +62,7 @@
 #define NAMES_DIR               "/apps/nautilus/desktop"
 #define HOME_NAME_KEY           "/apps/nautilus/desktop/home_icon_name"
 #define COMPUTER_NAME_KEY       "/apps/nautilus/desktop/computer_icon_name"
-#define MAX_ITEMS_OR_SUBMENU    5
+#define MAX_ITEMS_OR_SUBMENU    8
 #define MAX_BOOKMARK_ITEMS      100
 
 G_DEFINE_TYPE (PanelPlaceMenuItem, panel_place_menu_item, GTK_TYPE_IMAGE_MENU_ITEM)
